docs/support/troubleshooting.md

```diff
@@ -137,13 +137,14 @@ During building the following issue can occurs
 pkg_resources.extern.packaging.version.InvalidVersion: Invalid version: '0.23ubuntu1'
 ```
 
-The error is due to the fact that since version 66.0.0 `setuptools` enforces the python packages to be
+The error is due to the fact that for versions between 66.0.0 and 67.5.0 `setuptools` enforces the python packages to be
 [PEP-440](https://peps.python.org/pep-0440/) conformant.
+Since version 67.5.1 `setuptools` added [fallback](https://github.com/pypa/setuptools/commit/1640731114734043b8500d211366fc941b741f67) that makes it possible to work with old packages again.
```

Comment:
```suggestion
Since version 67.5.1 `setuptools` has a [fallback](https://github.com/pypa/setuptools/commit/1640731114734043b8500d211366fc941b741f67) that makes it possible to work with old packages again.
```

Reply:
Right, thanks for this correction